profiles/Anushka-codergirl.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/43162561/149078859-2fa22886-13fd-44d3-8a3a-6726bc8fc506.png",
```

Comment:
I recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/149189418-a2702a58-7abd-419d-b8d1-838fcae07e71.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/149189418-a2702a58-7abd-419d-b8d1-838fcae07e71.png",
```

Reply:
Done!